pkg/apis/prow/prow.go

```diff
@@ -1,6 +1,8 @@
 package prow
 
-import "time"
+import (
+	"time"
+)
```

Comment:
So if we are still blocked by the e2e test, this is really a noop and we could drop this file from the changelist.  Didn't notice it earlier and wouldn't hold this up if everything passed.

Reply:
I pulled it out of the changed list.  Interesting how vscode kept the parens.